gapic-generator/templates/default/service/client/method/docs/_request_normal.erb

```diff
@@ -1,25 +1,7 @@
 <%- assert_locals method -%>
-# @overload <%= method.name %>(request, options: nil)
-#   @param request [<%= method.request_type %> | Hash]
+# @param request [<%= method.request_type %> | Hash]
 <%- if method.doc_description -%>
-<%= indent method.doc_description, "#     " %>
-<%- end -%>
-#   @param options [Google::Gax::ApiCall::Options, Hash]
-#     Overrides the default settings for this call, e.g, timeout, retries, etc.
-#
-<%- arg_list = method.arguments.map { |arg| "#{arg.name}: nil"}.join ", " -%>
-# @overload <%= method.name %>(<%= arg_list %>, options: nil)
-<%- method.arguments.each do |arg| -%>
-#   @param <%= arg.name %> [<%= arg.doc_types %>]
-<%- if arg.doc_description -%>
-<%= indent arg.doc_description, "#     " %>
-<%- end -%>
-<%- end -%>
-#   @param options [Google::Gax::ApiCall::Options, Hash]
-#     Overrides the default settings for this call, e.g, timeout, retries, etc.
-<%- if method.arguments.find { |arg| arg.name == "options" } -%>
-#
-#     If you want to provide a {<%= method.request_type %>#options}
-#     value you will need to pass all attributes in a Hash instead of using
-#     named arguments.
+<%= indent method.doc_description, "#   " %>
 <%- end -%>
+# @param options [Google::Gax::ApiCall::Options, Hash]
```

Comment:
Add `nil` to the`options` types list?

Reply:
Since `nil` is the default value, we usually leave it off. Although we add "Optional."

Reply:
Added "Optional." to the `options` documentation.